ajax/libs/instagram-lite/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "instagram-lite",
+  "version": "2.0.0",
+  "filename": "instagramLite.min.js",
+  "description": "A simple, lightweight jQuery plugin used to display a user's Instagram photos.",
+  "keywords": [
+    "instagram",
+    "photos",
+    "api",
+    "feed"
+  ],
+  "license": "MIT",
+  "author": "Michael Lynch <mikethomaslynch@gmail.com>",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/michael-lynch/instagram-lite"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/michael-lynch/instagram-lite.git",
+    "fileMap": [
+      {
+        "basePath": "build",
+        "files": [
+          "**/*"
+        ]
+      },
+      {
+        "basePath": "src",
+        "files": [
+          "instragramLite*"
```

Comment:
The auto-update mechanism is cdnjs library [auto-update module](https://github.com/cdnjs/autoupdate)
It will take the git tags as each version from git repo.

We can use it locally, too.

If auto-update mechanism finds that there is not version folder, it will add the version folder and add files from the library repo.
The `files` of [auto-update config](https://github.com/cdnjs/cdnjs/blob/master/documents/autoupdate.md) is a pattern matcher allowing selection of multiple files by the auto-update mechanism.

Hope this comment will help you.

If you have any question, please let me know. Thank you.

Sorry that English is not my native language; please excuse typing errors.